code/modules/mob/living/simple_animal/abnormality/_tools/zayin/you_happy.dm

```diff
@@ -0,0 +1,122 @@
+#define STATUS_EFFECT_YMBH /datum/status_effect/display/YMBH
+/obj/structure/toolabnormality/you_happy
```

Comment:
Layer MUST be raised, it can current be obstructed making it IMPOSSIBLE to see your result